src/bicep/examples/newWorkload/README.md

```diff
@@ -11,23 +11,16 @@ Read on to understand what this example does, and when you're ready, collect all
 The docs on Azure virtual networking:  <https://docs.microsoft.com/en-us/azure/virtual-network/>.  This example deploys an additional [spoke network which is peered to the hub network](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview) of your MLZ instance. Additionally a few other items are deployed to enable connectivity in a secure manner:
 
 * A [route table is created and all external traffic is routed through the MLZ hub network](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-udr-overview)
-* While this example does not deploy an firewall, the target for all external traffic is the MLZ [Azure Firewall](https://docs.microsoft.com/en-us/azure/firewall/overview) hosted in the hub to ensure appropriate traffic filtering.
+* While this example does not deploy a firewall, the target for all external traffic is the MLZ [Azure Firewall](https://docs.microsoft.com/en-us/azure/firewall/overview) hosted in the hub to ensure appropriate traffic filtering.
 
 ## Pre-requisites
 
 1. A Mission LZ deployment (a deployment of mlz.bicep)
-2. Define some new values for required parameters described below.
-3. The outputs from a deployment of mlz.bicep (./src/bicep/examples/deploymentVariables.json).
-
-See below for information on how to create the appropriate deployment variables file for use with this template.
+1. Define values for required parameters described below, or generate a `deploymentVariables.json` file from the deployment (see below).
```

Comment:
```suggestion
1. Define values for required parameters described below, and generate a `deploymentVariables.json` file from the deployment (see below).
```

Should this be an "and" now? Curious because I don't know the behavior of `loadTextContent()` not finding a file at the path.